src/d3a/models/strategy/storage.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, risk: int=GeneralSettings.DEFAULT_RISK,
             InputProfileTypes.IDENTITY,
             StorageSettings.MIN_BUYING_RATE
         )
-        self.max_buying_rate_profile = {k: v[1] for k, v in break_even.items()}
+        self.max_buying_rate_profile = {k: v[0] for k, v in break_even.items()}
```

Comment:
It is ok for now, but in D3ASIM-968 I have already added a more future-proof fix for that which does not use a tuple, but a namedtuple instead. That way it will be easy for the reader to understand which one is the buy and the sell break even point.